Source/automap.cpp

```diff
@@ -218,51 +217,51 @@ void __cdecl DrawAutomap()
 	mapy = MapY - 1;
 
 	if (cells & 1) {
-		screen_x = 384 - AutoMapPosBits * ((cells - 1) >> 1);
-		screen_y = 336 - AutoMapXPos * ((cells + 1) >> 1);
+		sx = 384 - AutoMapPosBits * ((cells - 1) >> 1);
+		sy = 336 - AutoMapXPos * ((cells + 1) >> 1);
 	} else {
-		screen_x = 384 - AutoMapPosBits * (cells >> 1) + AutoMapXPos;
-		screen_y = 336 - AutoMapXPos * (cells >> 1) - AutoMapYPos;
+		sx = 384 - AutoMapPosBits * (cells >> 1) + AutoMapXPos;
+		sy = 336 - AutoMapXPos * (cells >> 1) - AutoMapYPos;
 	}
 	if (ViewX & 1) {
-		screen_x -= AutoMapYPos;
-		screen_y -= AMPlayerX;
+		sx -= AutoMapYPos;
+		sy -= AMPlayerX;
 	}
 	if (ViewY & 1) {
-		screen_x += AutoMapYPos;
-		screen_y -= AMPlayerX;
+		sx += AutoMapYPos;
+		sy -= AMPlayerX;
 	}
 
-	screen_x += AutoMapScale * ScrollInfo._sxoff / 100 >> 1;
-	screen_y += AutoMapScale * ScrollInfo._syoff / 100 >> 1;
+	sx += AutoMapScale * ScrollInfo._sxoff / 100 >> 1;
+	sy += AutoMapScale * ScrollInfo._syoff / 100 >> 1;
 	if (invflag || sbookflag) {
-		screen_x -= 160;
+		sx -= 160;
 	}
 	if (chrflag || questlog) {
-		screen_x += 160;
+		sx += 160;
 	}
 
 	for (i = 0; i <= cells + 1; i++) {
-		int x = screen_x;
+		int x = sx;
 		int y;
 
 		for (j = 0; j < cells; j++) {
 			WORD maptype = GetAutomapType(mapx + j, mapy - j, TRUE);
 			if (maptype)
-				DrawAutomapType(x, screen_y, maptype);
+				DrawAutomapType(x, sy, maptype);

```

Comment:
Use `sx` instead of `x` in this call. It is still valid and equivalent, since `x = sx` at line 245.

Reply:
Unfortunatly that turns out not to be bin exact.

Reply:
Oh, well. Then keep it as is.